client/src/components/requester_task/task.html

```diff
@@ -1,8 +1,8 @@
 <div id="requester-task" ng-controller="RequesterTaskCtrl as ctrl">
   <nav-bar></nav-bar>
   
-  <div ui-view="location"></div>
-  <div ui-view="item"></div>
-  <div ui-view="time_price"></div>
-  <div ui-view="confirm"></div>
+  <div ng-show = "ctrl.location" ui-view="location"></div>

```

Comment:
no space before and after = sign
should be `ng-show="ctrl.location"`

also on line 5, 6, 7